java/main/src/main/java/com/saucelabs/saucebindings/GitManager.java

```diff
@@ -0,0 +1,24 @@
+package com.saucelabs.saucebindings;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+public class GitManager {
+    private static String currentBranch = "_default_";
+
+    public static String getCurrentBranch() {
+        try {
+            Process process = Runtime.getRuntime().exec("git rev-parse --abbrev-ref HEAD");
+            process.waitFor();
+            BufferedReader reader = new BufferedReader(new InputStreamReader(process.getInputStream()));
+            String line = reader.readLine();
+            if (!line.contains("fatal:")) {
+                currentBranch = line;
```

Comment:
should the line be trimmed?

Reply:
What do you mean? (and I don't remember what all I did to get while trying to get this to work when I wrote it over a year ago).

Reply:
I mean whether it could be that `line` ends with a line break. Also, I would assume it would be safer to check the process exit code than the actual stdout